change/@fluentui-web-components-518b3512-c7f4-4073-91e2-880c782b0864.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "revert design token syntax change for DX",
+  "packageName": "@fluentui/web-components",
+  "email": "=",
+  "dependentChangeType": "patch"
```

Comment:
should be minor not patch right? If folks adopted the new behavior this is breaking

Reply:
This specifically is for the `dependentChangeType`, which I believe is for dependents in monorepos. The version will still use `prerelease`, but I'm not aware of a way to force that to be `minor` (agree it should). Using the CLI I can only specify `prerelease`.

Reply:
@justslone @spmonahan I confirmed with Martin that we need to stay on the prerelease.

Reply:
🚢ship it then? (maybe on Monday would be a good idea)